packages/web/src/components/createCell.tsx

```diff
@@ -91,7 +97,9 @@ type Guaranteed<T> = {
  * post.id // post is non optional, so no need to do post?.id
  *
  */
-export type CellSuccessData<TData = any> = Omit<Guaranteed<TData>, '__typename'>
+export type CellSuccessData<TData = any> = ConditionallyGuaranteed<
+  Omit<TData, '__typename'>
+>
```

Comment:
We pass `TData` to `ConditionallyGuaranteed` with `__typename` already omitted so we don't have to account for it there.